app/models/mixins/ui_service_mixin.rb

```diff
@@ -0,0 +1,24 @@
+module UiServiceMixin
+  def icons
+    {
+      :ContainerReplicator => {:type => "glyph", :icon => "\uE624", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-replicator
+      :ContainerGroup      => {:type => "glyph", :icon => "\uF1B3", :fontfamily => "FontAwesome"},             # fa-cubes
+      :ContainerNode       => {:type => "glyph", :icon => "\uE621", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-container-node
+      :ContainerService    => {:type => "glyph", :icon => "\uE61E", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-service
+      :ContainerRoute      => {:type => "glyph", :icon => "\uE625", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-route
+      :Container           => {:type => "glyph", :icon => "\uF1B2", :fontfamily => "FontAwesome"},             # fa-cube
+      :Host                => {:type => "glyph", :icon => "\uE600", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-screen
+      :Vm                  => {:type => "glyph", :icon => "\uE90f", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-virtual-machine
```

Comment:
Not for this PR, but if we are going to be embedding strings like this, perhaps there should be constants defined somewhere (or a method that does a lookup)...in this way you also wouldn't need the comment, and the code would be self-documenting...

Thinking something like the following

```ruby
FONTICONS = {
  "fa-cubes"              => "\uF1B3",  
  "pficon-container-node" => "\uE621",
  "pficon-replicator"     => "\uE624",
  ...
}.freeze

# caller

def icons
  {
    :ContainerReplicator => {:type => "glyph", :icon => FONTICONS["pficon-replicator"],     :fontfamily => "PatternFlyIcons-webfont"},
    :ContainerGroup      => {:type => "glyph", :icon => FONTICONS["fa-cubes"],              :fontfamily => "FontAwesome"},
    :ContainerNode       => {:type => "glyph", :icon => FONTICONS["pficon-container-node"], :fontfamily => "PatternFlyIcons-webfont"},
    ...
  }
```

Perhaps even the fontfamily can move into that constant, since I don't think it can change.